VERSION

```diff
@@ -1 +1 @@
-0.1.1
+0.2.0
```

Comment:
this feels like more of a dot release than a minor release, no?

Reply:
Not that it really matters

Reply:
I think it's a breaking change because URLs that "worked" with 0.1.1 won't work with this release

And I made a minor release bump for what I think is a breaking change because the rules for semantic versioning make a special exception when you're on the 0.x series

Reply:
ah, gotcha. Makes sense.

Reply:
but I don't feel strongly about that so if you do, I'll change it

Reply:
That makes sense. Thank you for the explanation @jameslamb

Reply:
nope. I care zero amount. Just wasn't thinking about the change in terms of breaking vs. non-breaking. That logic makes sense to me. (As I side-note, I have never worked on a team that did anything other than fully arbitrary versioning, so I was not aware of the particular rules around that)

Reply:
> full arbitrary versioning

haha I love that term! Ok cool, I'll keep it as 0.2.0 then.